pkg/installer/manager.go

```diff
@@ -20,7 +20,8 @@ type manager struct {
 	log *logrus.Entry
 	env env.Interface
 
-	doc             *api.OpenShiftClusterDocument
+	uuid            string
```

Comment:
Is there a more descriptive name for `uuid`? perhaps docId, or clusterDocId? I know we want to remove reliance on the doc, but what is this `uuid` for then?